index.html

```diff
@@ -0,0 +1,273 @@
+                        </p>
+                        <ul class="art_print_btn_container">
+                            <li><button class="transparent-btn">HTML</button></li>
+                            <li><button class="transparent-btn">Bootstrap</button></li>
+                            <li><button class="transparent-btn">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <div class="art-main-btn-container">
+                        <button class="primarybtn">See Project</button>
+                    </div>
+                </div>
+
+                <div class="professional_art_printing-section-items">
+                    <div class="art_print_container">
+                        <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                        <p class="art_print_paragraph">
+                            A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul class="art_print_btn_container">
+                            <li><button class="transparent-btn">HTML</button></li>
+                            <li><button class="transparent-btn">Bootstrap</button></li>
+                            <li><button class="transparent-btn">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <div class="art-main-btn-container">
+                        <button class="primarybtn">See Project</button>
+                    </div>
+                </div>
+
+                <div class="professional_art_printing-section-items">
+                    <div class="art_print_container">
+                        <h2 class="art_print_header">Profesional Art Printing Data</h2>
+                        <p class="art_print_paragraph">
+                            A daily selection of privately personalized reads; no accounts or
+                            sign-ups required. Has been the industry's standard.
+                        </p>
+                        <ul class="art_print_btn_container">
+                            <li><button class="transparent-btn">HTML</button></li>
+                            <li><button class="transparent-btn">Bootstrap</button></li>
+                            <li><button class="transparent-btn">Ruby</button></li>
+                        </ul>
+                    </div>
+                    <div class="art-main-btn-container">
+                        <button class="primarybtn">See Project</button>
+                    </div>
+                </div>
+            </section>
+        </section>
+    </section>
+
+    <!-- DARK SECTION -->
+    <section class="dark_section">
+        <!-- ABOUT-ME SECTION -->
+        <section class="aboutme_section">
+            <div class="aboutme_container">
+                <h1 class="aboutme_header_text">About me</h1>
```

Comment:
- [ ] great job. Everything is on point. Please kindly change the `about me` from h1 tag to h2 or h3. its good to have just one H1 on the page to indicate what the entire web page is about